src/main/java/org/polyfill/components/Polyfill.java

```diff
@@ -73,12 +73,11 @@ public String getBrowserRequirement(String browserName) {
         return getListFromMap(this.polyfillMap, DEPENDENCIES_KEY);
     }
 
-    @Nullable
+
     public String getRawSource() {
         return getStringFromMap(this.polyfillMap, RAW_SOURCE_KEY);
     }
 
-    @Nullable
     public String getMinSource() {
```

Comment:
this method shouldn't return null, if there isn't source available it should return empty string "";